pytorch_lightning/metrics/classification.py

```diff
@@ -52,22 +52,25 @@ class Accuracy(TensorMetric):
     def __init__(
         self,
         num_classes: Optional[int] = None,
-        reduction: str = "elementwise_mean",
+        class_reduction: str = 'micro',
         reduce_group: Any = None,
     ):
         """
         Args:
             num_classes: number of classes
-            reduction: a method to reduce metric score over labels (default: takes the mean)
-                Available reduction methods:
-                - elementwise_mean: takes the mean
-                - none: pass array
-                - sum: add elements
+            class_reduction: reduction method for multiclass problems
```

Comment:
```suggestion
            class_reduction: a method to reduce metric score over labels
```
only for multiclass? Just to avoid any confusion for anyone looking at the docs, I think this was better. But if it's ok then all good :+1:

Reply:
It is also used for binary cases, however in that case I think that some of the reduction methods will produce the same result. I will change the text everywhere to your suggestion. Thanks 👍

Reply:
Also, should similar changes be made in `stat_scores_multiple_classes` for class_reduction?